projects/telemetry/src/gps.c

```diff
@@ -0,0 +1,78 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+#include "status.h"
+#include "gps.h"
+#include "gpio.h"
+#include "uart_mcu.h"
+#include "misc.h"
+#include "nmea.h"
+
+// How many handlers can we possibly need?
+#define GPS_HANDLER_ARRAY_LENGTH 5
+
+static const UARTPort port = UART_PORT_2;
+static GPSHandler gps_handler[GPS_HANDLER_ARRAY_LENGTH] = {0};
+static GGAHandler gga_handler[GPS_HANDLER_ARRAY_LENGTH] = {0};
+static uint32_t init = 0;
```

Comment:
Nit: Static init conditions are usually a bad idea. It makes resetting things for testing very unpredictable. Ensure you wipe all your static memory to 0 in the init function then re-run any config you need to do also make it a bool since 32 bits is overkill.